Popup/Popup.module.less

```diff
@@ -14,12 +14,12 @@
 		height: auto;
 	}
 
-	&.centered {
+	.buttons {
 		text-align: center;
+	}
 
-		.buttons {
-			text-align: center;
-		}
+	&.centered {
+		text-align: center;

```

Comment:
![image](https://user-images.githubusercontent.com/34676178/116952594-faba9a00-acc5-11eb-9ca5-f2d08fc6f480.png)
I don't know the reason, but there is side effect about text-align on other skin

Reply:
Hello, we created PLAT-139055 for this matter. It only waits for this PR to be merged